README.md

```
@@ -10,3 +10,45 @@ is what I want for my job, so that's what it is going to be!
 The first db will be postgresql, because that is what we currently use at my company.
 After that, sqllite will be added, then mongodb. Then it will be opened up to the
 public for any other db that people want to add.
+
+TLDR: ONLY WORKS ON POSTGRES
+
+steps to use:
+1. clone the repo
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)